frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -589,14 +594,25 @@ export default Vue.extend({
         if (this.enabled_edit_schema == true) {
             this.edit_ui_schema()
         }
+        this.hotkey_listener = HotkeyListener.getInstance()
+
+        this.hotkey_listener.addFilter(() => {
```

Comment:
Hey @anthony-sarkis , please note that I added `hotkey_listener.addFilter` method that allows us to use current logic (the one that updates `this.$store.state.user.is_typing_or_menu_open`), to control firing of the hotkey handlers. Please see `HotkeyListener` definition for more details.